src/libraries/IterableMap.sol

```diff
@@ -0,0 +1,104 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/Controller.sol';
+
+contract IterableMap {
```

Comment:
This is a delegated contract, which means it _must_ derive from `DelegationTarget` as its first derivation class:
```
contract IterableMap is DelegationTarget { ... }
```